Examples/AmplitudeSwiftUIExample/AmplitudeSwiftUIExample/ExamplePlugins/WebViewSyncPlugin.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  WebViewSyncPlugin.swift
+//  AmplitudeSwiftUIExample
+//
+//  Created by Chris Leonavicius on 1/9/25.
+//
+
+import AmplitudeSwift
+import ObjectiveC
+import WebKit
+
+class WebViewSyncPlugin: NSObject, Plugin {
+
+    let type = PluginType.utility
+
+    private let webviews = NSHashTable<WKWebView>.weakObjects()
+
+    private static let swizzleWebViewInitializer: Void = {
```

Comment:
Instead of swizzling the initializer which would affect all web views, would it be better to document that consumers could attach `WebViewSyncPlugin.registerWebview(webview)` at their own callsites to attach the Amplitude plugin? 

This would allow us to specify if we wish to attach the user script for a particular webview rather than automatically do it for all, as well as avoid any downsides of the swizzling approach.